src/pages/Cart/Cart.jsx

```diff
@@ -1,46 +1,60 @@
 import React from "react";
 import "./cart.css";
+import { useCart } from "../../context";
 import { CartCard } from "./components";
-
-const data = {
-  id: 1,
-  brand: "sony",
-  name: "DualSense Gamepad",
-  price: 5999,
-  discountedPrice: 4599,
-  img: "https://rukminim1.flixcart.com/image/416/416/kzfvzww0/gamepad/b/u/i/-original-imagbg7nnjpyyrzh.jpeg?q=70",
-  category: "Gaming Accessories",
-  rating: 5,
-};
+import cartEmpty from "../../assets/fallback/empty-cart.png";
+import {
+  getCartPrice,
+  getCartTotal,
+  getDiscountedTotal,
+} from "../../utils/generalUtils";
 
 export function Cart() {
+  const {
+    cartState: { data: cartItems },
+  } = useCart();
+  const cartPrice = getCartPrice(cartItems);
+  const cartDiscount = getDiscountedTotal(cartItems);
+  const cartTotal = getCartTotal(cartItems);
```

Comment:
Your Quantity is increasing, but you have updated the cart price details as per quantity. If it will be done in future PR's. Then great.

Reply:
Yeah. I didn't think about this will fix this now.